docs/build-transaction-construction.md

```diff
@@ -0,0 +1,210 @@
+---
+id: build-transaction-construction
+title: Transaction Construction and Signing
+sidebar_label: Transaction Construction
+---
+
+This page will discuss the transaction format in Polkadot and how to create, sign, and broadcast
+transactions. Like the other pages in this guide, this page demonstrates some of the available
+tools. **Always refer to each tool's documentation when integrating.**
+
+## Transaction Format
+
+Polkadot has some basic transaction information that is common to all transactions.
+
+- Address: The SS58-encoded address of the sending account.
+- Block Hash: The hash of the [checkpoint](build-protocol-info#transaction-mortality) block.
+- Block Number: The number of the checkpoint block.
+- Genesis Hash: The genesis hash of the chain.
+- Metadata: The SCALE-encoded metadata for the runtime when submitted.
+- Nonce: The nonce for this transaction.\*
```

Comment:
What is this \* for?  Along with the one on line 26?

Reply:
> Along with the one on line 26?

yes